### DISCUSSION TITLE
[FEATURE]创建静态网站的时候，除了80端口能用，其他端口为啥就不行，而且443在安装之后设置时显示被OpenResty占用

### DISCUSSION THREAD

**Discussion by lobmoo** [1Panel-dev/1Panel#4359] [Q&A]
### 1Panel 版本

v1.10.2-lts

### 请描述您的需求或者改进建议

创建静态网站的时候，除了80端口能用，其他端口为啥就不行，而且443在安装之后设置时显示被OpenResty占用 
![image](https://github.com/1Panel-dev/1Panel/assets/45560941/9869ab21-c66a-4cef-832d-a64eb5797287)

**Comment by zhengkunwang223** [comment]
如果是其他端口都被占用 考虑是不是安装了 类似 虚拟机性能优化工具 的插件
有用户反馈，深信服超融合装了“虚拟机性能优化工具”后所有端口都显示被占用

**Reply by lobmoo** [reply]
我的是腾讯云，ufw我开了我想配置的端口，443占用先不说，就是我配置的8884端口，我确认端口开了，域名配置的就是IP  想直接ip访问，用netstat查看，docker和宿主机的该端口都监听了，但是就是访问不了，同样的操作，80端口就可以。  我本想试试443端口，发现占用。我是刚开的腾讯云，系统干净，panel是第一个应用，应该没啥问题

**Reply by zhengkunwang223** [reply]
> 我的是腾讯云，ufw我开了我想配置的端口，443占用先不说，就是我配置的8884端口，我确认端口开了，域名配置的就是IP  想直接ip访问，用netstat查看，docker和宿主机的该端口都监听了，但是就是访问不了，同样的操作，80端口就可以。  我本想试试443端口，发现占用。我是刚开的腾讯云，系统干净，panel是第一个应用，应该没啥问题

你是什么类型的网站，反向代理还是静态网站，反向代理的话，后端是什么服务，端口是多少？

**Reply by lobmoo** [reply]
我就是一个静态网页，做测试的，直接访问到html文件，没有做反代，，端口6666.  除了80  其他的我都试了，都不可以

**Reply by lobmoo** [reply]
![image](https://github.com/1Panel-dev/1Panel/assets/45560941/ad361838-6a2b-41bf-84a5-7c6f6242bd30)
![image](https://github.com/1Panel-dev/1Panel/assets/45560941/2cb05a7c-9fe0-4140-ba36-9c5dffcef7d7)

配置如上

**Reply by wanghe-fit2cloud** [reply]
通过 docker run 命令在本地手动起一个 nginx 服务，端口 6666，确认一下能不能正常访问吧。

**Reply by lobmoo** [reply]
端口问题我通过反代的方式去解决了，，但是443被占用好像更难受，拉代码的时候，需要443端口，比如go，所以这个不知道有没有方案
![image](https://github.com/1Panel-dev/1Panel/assets/45560941/f1649e51-1125-4f5f-a69f-e6cbe06ac730)

**Reply by lobmoo** [reply]
补充一下，80开了对外域名和子域名反代，在用，但是443没有挂任何服务，插件装完就被占用了

**Comment by anand-lyn** [comment]
所以这个问题解决了没，nginx 不同server是可以复用端口的啊，我创建网站指定不同域名 指定ssl 443端口为啥报openresty端口占用

**Reply by lobmoo** [reply]
后来换了端口  没有解决
